packaging/installer/netdata-updater.sh

```diff
@@ -532,7 +536,9 @@ update_build() {
   if update_available; then
     download "${NETDATA_TARBALL_CHECKSUM_URL}" "${ndtmpdir}/sha256sum.txt" >&3 2>&3
     download "${NETDATA_TARBALL_URL}" "${ndtmpdir}/netdata-latest.tar.gz"
-    if [ -n "${NETDATA_TARBALL_CHECKSUM}" ] && grep "${NETDATA_TARBALL_CHECKSUM}" sha256sum.txt >&3 2>&3; then
+    if [ -n "${NETDATA_TARBALL_CHECKSUM}" ] &&
+      grep "${NETDATA_TARBALL_CHECKSUM}" sha256sum.txt >&3 2>&3 &&
+      [ "$NETDATA_FORCE_UPDATE" != "1" ]; then
```

Comment:
This should probably be in the `update_available` function, otherwise this code will never actually be reached.

Reply:
It is in the function, but that is an additional check where we compare `"${NETDATA_LIB_DIR}/netdata.tarball.checksum"` with the version checksum

https://github.com/netdata/netdata/blob/1023c3e196fafcc21c10bf3c3c95b88339b82f1a/packaging/installer/netdata-updater.sh#L510-L514

Reply:
Ah, you’re right, I somehow missed that.